web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -578,6 +580,56 @@ function ManageSubscriptionModal({
   )
 }
 
+function Plan({
+  className,
+  name,
+  cost,
+  features,
+  startButton,
+  highlight,
+}: {
+  className?: string
+  name: React.ReactNode
+  cost: React.ReactNode
+  features: React.ReactNodeArray
+  startButton: React.ReactNode
+  highlight?: boolean
+}) {
+  return (
+    <Card
+      className={
+        "shadow-sm h-100 " + className + (highlight ? " shadow " : "")
+      }>
+      <Card.Header>
+        <h4 className="text-center">{name}</h4>
+      </Card.Header>
+      <Card.Body className="d-flex flex-column">
+        <h1 className="text-center">{cost}</h1>
+        <div className="flex-grow-1 d-flex flex-column">
+          <ul className="flex-grow-1 list-unstyled mt-3 mb-4 text-center">
+            {features.map(x => (
+              <li>{x}</li>
```

Comment:
probably doesn't matter since we aren't reordering the elements or anything, but seems like we don't have eslint-react setup